aws/table_aws_wellarchitected_lens_review_improvement.go

```diff
@@ -0,0 +1,219 @@
+package aws
+
+import (
+	"context"
+	"errors"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected"
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected/types"
+	"github.com/aws/smithy-go"
+
+	wellarchitectedv1 "github.com/aws/aws-sdk-go/service/wellarchitected"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsWellArchitectedLensReviewImprovement(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_wellarchitected_lens_review_improvement",
+		Description: "AWS Well-Architected Lens Review Improvement",
+		List: &plugin.ListConfig{
+			ParentHydrate: listWellArchitectedWorkloads,
+			Hydrate:       listWellArchitectedLensReviewImprovements,
+			// TODO: Uncomment and remove extra check in
+			// listWellArchitectedLensReviewImprovements function once this works again
+			// IgnoreConfig: &plugin.IgnoreConfig{
+			// 	ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ResourceNotFoundException", "ValidationException"}),
+			// },
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "workload_id", Require: plugin.Optional},
+				{Name: "lens_alias", Require: plugin.Optional},
+				{Name: "milestone_number", Require: plugin.Optional},
```

Comment:
Should we not include `pillar_id` as an optional parameter?

https://docs.aws.amazon.com/wellarchitected/latest/APIReference/API_ListLensReviewImprovements.html

Reply:
Added pillar_id as an optional quals.